core/cli/node/wallet.hpp

```diff
@@ -0,0 +1,380 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "api/rpc/json.hpp"
+#include "cli/node/node.hpp"
+#include "cli/validate/address.hpp"
+#include "codec/json/json.hpp"
+#include "common/file.hpp"
+#include "common/hexutil.hpp"
+#include "crypto/signature/signature.hpp"
+#include "primitives/big_int.hpp"
+#include "storage/ipfs/api_ipfs_datastore/api_ipfs_datastore.hpp"
+#include "vm/state/impl/state_tree_impl.hpp"
+
+namespace fc::cli::_node {
```

Comment:
Why `_node`?

Reply:
Because tree of command build in fc::cli::_node (see core/cli/node/main.cpp)

Reply:
Names that starts with underscore are reserved. It is misleading and violates naming rules. Please, consider
```suggestion
namespace fc::cli::node {
```

Reply:
@Alexey-N-Chernyshov, namespace `_node` (fc::cli::_node) was used to prevent shadowing `node` namespace (outside of fc::cli).  
Instead of `node`, `cli_node` or `node_cli` would be better.

Reply:
Fixed on cli_node